packages/debugger/lib/controller/sagas/index.js

```diff
@@ -211,6 +212,22 @@ function* stepOver() {
   );
 }
 
+/**
+ * runToEnd - run the debugger till the end
+ */
+function* runToEnd() {
+  var finished;
```

Comment:
Never use `var`!
```suggestion
  let finished;
```

Reply:
We prefer `let` and `const` for variable declaration because it makes the code easier to reason about. With `var` you have to be aware of scoping and other implications of how the interpreter splits variable declaration from initialization. `let` and `const` define blocked scoped local variables and are easier to reason about. There's even annoying global pollution with `var` that thankfully `let` doesn't allow. See: [Hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) and [let statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) for more information.


Truffle's codebase has lots of `var` declarations because it was written before `let` and `const` came into the language. Now that they are available we should use them.

Reply:
Thanks @cds-amal . I will keep this thing in mind while writing codes.